### PR TITLE
Dana katzenelson validate pat test

### PR DESCRIPTION
**Pull request by dekatzenel**
### Background

Misc SDL test file improvements, in parallel with those in p-e

### Changes

* <Describe changes here>

### Testing

* <Testing steps>

### CODE REVIEW

**Review by lindsey-w — APPROVED** [review]
LGTM 👍🏻

**Comment by dekatzenel** [comment]
I'll merge once the p-e changes are merged

**Review by maxrichie5 — APPROVED** [review]
Seems pretty cool to me

**Pull request merged**
